2015-10-27.md

```diff
@@ -0,0 +1,32 @@
+Hi Tesselators!
+
+It's been a few weeks since the last This Week in Tessel, our apologies! We've been hard at work on the Tessel repos, building and growing the project.
+
+### Shipping Timeline
+After countless back and forths with our FCC testing counterparts, we have an end in sight. Seeed Studio expects us to have results from FCC testing back by Friday, November 27th. Once FCC certification is passed, sourcing the components will take up to 6 weeks. Then we can start producing in batches of 500/week. This puts the first shipments out in mid-January. Feel free to get in touch if the new date is an issue for you (support@tessel.io).
```

Comment:
Don't make it out to seem like you're duking it out, the FCC has a super important function even if it's frustrating (and expensive?) to deal with their bureaucracy